### ISSUE TITLE
Error importing react-modal

### ISSUE THREAD

**Issue by mrscobbler** [reactjs/react-modal#418]
### Summary:
I tried to add react-modal to my project and I'm getting this error:

```
Module parse failed: /node_modules/react-modal/lib/components/Modal.js Unexpected token (36:19)
You may need an appropriate loader to handle this file type.
| 
|   /* eslint-disable react/no-unused-prop-types */
|   static propTypes = {
|     isOpen: PropTypes.bool.isRequired,
|     style: PropTypes.shape({
```

### Steps to reproduce:

 1. `yarn add react-modal`
 2. Add this statement at the top of my code: `import Modal from 'react-modal'`
 

### Expected behavior:
Modal should not cause an error when importing

### Additional notes:
I tried deleting my node_modules folder and running `yarn` but that didn't seem to fix the issue

```

**Comment by ingro** [comment]
Yeah it seems like the code is not compiled correctly.

**Comment by theWaR13** [comment]
Same for me.

**Comment by diasbruno** [comment]
hmm, I thought `babel-preset-stage-2` was enough for this.
A PR to fix this would change from `stage-2` to `latest`.
I'll be back in a few minutes.

**Comment by danhayden** [comment]
Is this not because `main` in package.json is `"./lib/index.js"` and should be `"./dist/index.js"`?

As importing the module is currently returning the source code and not the transpiled code.

**Comment by diasbruno** [comment]
Sorry for the inconvenience. I just had to push forward to get `v2.0.0` and fix our repository, but I probably forgot to push an important commit from the old master (not branch `v2`).
@danhayden you are right.

See PR #417, @ajfuller is already working on this.

Please, use `v1.9.7` until we fix this issue.

Thank you all!

**Comment by LeG3nDz** [comment]
Same issue, but work with `v1.9.7` in Meteor project.

**Comment by diasbruno** [comment]
Released `v2.0.1`.

**Comment by LeG3nDz** [comment]
Fix for me, but with warning after update to **2.0.1** :

`Warning: Invalid argument supplid to oneOfType. Expected an array of check functions, but received undefined at index 0.`

**Comment by LeG3nDz** [comment]
Fix for me. Thank you @diasbruno  !

**Comment by diasbruno** [comment]
@LeG3nDz Thank you. Also, a fix to removed all `prop-types` warnings was released in `v2.0.2`.

**Comment by LeG3nDz** [comment]
@diasbruno Yep, `prop-types` warning are removed in `v2.0.2`.

Thank you ;)
